contrib/gitian-descriptors/gitian-win-old.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: "zcoin"
+suites:
+- "trusty"
+architectures:
+- "amd64"
+packages:
+- "mingw-w64"
+- "g++-mingw-w64"
+- "git-core"
+- "unzip"
+- "nsis"
+- "faketime"
+- "autoconf2.13"
+- "libtool"
+- "automake"
+- "pkg-config"
+- "bsdmainutils"
+
+reference_datetime: "2013-06-01 00:00:00"
+remotes:
+- "url": "https://github.com/zcoinofficial/zcoin.git"
+  "dir": "zcoin"
+  "commit": "sync-fix-0.8.7.7"
```

Comment:
i was testing in this branch, but the right one is the master or dont use this line